examples/instantmesh/train.py

```diff
@@ -0,0 +1,460 @@
+""" InstantMesh Stage-1 Training Script """
```

Comment:
rename to train_stage1.py?

Reply:
The `train.py` is for both stage 1 and stage 2 training. The difference in stage 2 is in the renderer side, where a flexicube trainable isosurface extraction is included to incorporate additional depth, normal, and regularization loss. Due to the incapability of operators like `mint.unique()` and cuda extensions, stage 2 is not included yet.

Reply:
This confusing comment snippet is removed. Thanks.